src/main/java/com/crud/tasks/trello/client/TrelloClient.java

```diff
@@ -0,0 +1,52 @@
+package com.crud.tasks.trello.client;
+
+import com.crud.tasks.domain.TrelloBoardDto;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+@Component
+public class TrelloClient {
+    @Value("${trello.api.endpoint.prod}")
+    private String trelloApiEndpoint;
+
+    @Value("${trello.app.key}")
+    private String trelloAppKey;
+
+    @Value("${trello.app.token}")
+    private String trelloToken;
+
+    @Value("${trello.app.username}")
+    private String trelloUsername;
+
+    @Autowired
```

Comment:
lepiej wstrzyknąć to oraz powyższe pola w momencie tworzenia obiektu - w konstruktorze